Formula/rpki-client.rb

```diff
@@ -0,0 +1,55 @@
+class RpkiClient < Formula
+  desc "OpenBSD portable rpki-client"
+  homepage "https://www.rpki-client.org/index.html"
+  url "https://ftp.openbsd.org/pub/OpenBSD/rpki-client/rpki-client-7.4.tar.gz"
+  sha256 "efd336137707ad73fa51f8480cff686c21fea7086619432069c1a02553b68d57"
+  license "ISC"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on arch: :x86_64
+  depends_on "libressl"
+  depends_on macos: :mojave
```

Comment:
Did you actually verify that it fails to build on older versions?

Reply:
only that CI failed.

Reply:
But we don't run CI on versions older than Catalina, so I'm not sure what CI failure you're referring to that justifies this.

Reply:
I don't know what I saw. probably got confused. I've removed this and put depends_on :macos to stop Ubuntu only for now.